AppiumForMac/Server/Controller/AfMSessionController.m

```diff
@@ -891,6 +891,55 @@ -(PFUIElement*) windowForHandle:(NSString*)windowHandle
 	return [windows objectAtIndex:windowIndex];
 }
 
+-(BOOL) scrollPage:(id)element
+{
+    float increment_by = 0.25;
```

Comment:
could you please use camelCase naming to match with the current project coding style?